sw/device/lib/testing/test_framework/ottf_isrs.c

```diff
@@ -10,6 +10,56 @@
 #include "sw/device/lib/runtime/ibex.h"
 #include "sw/device/lib/runtime/log.h"
 
+// Fault reasons from
+// https://riscv.org/wp-content/uploads/2017/05/riscv-privileged-v1.10.pdf
+static const char *exception_reason[] = {
+    "Instruction Misaligned",
+    "Instruction Access",
+    "Illegial Instruction",
+    "Breakpoint",
+    "Load Address Misaligned",
+    "Load Access Fault",
+    "Store Address Misaligned",
+    "Store Access Fault",
+    "U-mode Ecall",
+    "S-mode Ecall",
+    "Reserved",
```

Comment:
Is it worth numbering the reserved interrupts? E.g. "Reserved (N)" where N is the interrupt number? If for some reason they occur in practice it might be useful to be able to tell at a glance what interrupt it is.

Reply:
The optimistic view is that it would help people... the pessimistic view is it would help people to do what they shouldn't and use reserved faults.

I think if we want to reveal the exception code it should be done more directly as an exception code and that mapping to something more specific than the cited spec is a risk or a bad habit.